code/modules/mob/living/carbon/human/human_powers.dm

```diff
@@ -559,4 +559,33 @@
 		if (G.affecting)
 			G.affecting.forceMove(locate(T.x + rand(-1,1), T.y + rand(-1,1), T.z))
 		else
-			qdel(G)
\ No newline at end of file
+			qdel(G)
+
+/mob/living/carbon/human/proc/pry_open(obj/machinery/door/A in filter_list(oview(1), /obj/machinery/door))
```

Comment:
No point adding another proc to do something we already have a proc for, this can be converted into a typecache + `typecache_filter_list()` (which should be faster).